docs/concepts/infrastructure.md

```diff
@@ -183,8 +183,9 @@ Requirements for `DockerContainer`:
 | auto_remove | Bool indicating whether the container will be removed on completion. If False, the container will remain after exit for inspection. |
 | command | A list of strings specifying the command to run in the container to start the flow run. In most cases you should not override this. |
 | env	| Environment variables to set for the container. |
-| image | An optional string specifying the tag of a Docker image to use. Defaults to the Prefect image. |
+| image | An optional string specifying the tag of a Docker image to use. Defaults to the Prefect image. If the image is stored anywhere other than a public Docker Hub registry, the image name should begin with the registry address. |
```

Comment:
```suggestion
| image | An optional string specifying the tag of a Docker image to use. Defaults to the Prefect image. If the image is stored anywhere other than a public Docker Hub registry, use a corresponding registry block, e.g. `DockerRegistry` or ensure otherwise that your execution layer is authenticated to pull the image from the image registry. |
```
I think this related to the previous implementation, so we may reword that